perses/rjmc/geometry.py

```diff
@@ -226,6 +227,11 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
             # DEBUG: Write growth stages
             from simtk.openmm.app import PDBFile
             prefix = '%s-%d-%s' % (self.pdb_filename_prefix, self.nproposed, direction)
+            self._proposal_pdbfile = open("%s-proposal.pdb" % prefix, 'w') # PDB file for proposal probabilities
```

Comment:
I'm curious if we can just write out the sequence of atoms in order. That way, to visualize the introduction, we'd only need to make them visible in the correct order.